spec/orangelight/cjk_mapping_spec.rb

```diff
@@ -83,6 +83,13 @@ def add_single_field_doc char
         params = {qf: 'cjk_title', pf: 'cjk_title', q: '长沙走马楼三国吴简'}
         expect(solr_resp_doc_ids_only(params)).to include('1')
       end
+      it "can find notes" do
```

Comment:
Did this test fail without these changes?

Reply:
Yes.